### ISSUE TITLE
Rule request: Prefer non-alias of `ActiveRecord::Base.update` and friends

### ISSUE THREAD

**Issue by rhys-vdw** [bbatsov/rubocop#4208]
I'd like a rule that enforces consistent use of either `update` and `update!` or their aliases `update_attributes` and `update_attributes!`.

Eg:

```rb
# good
record.update!
return record if record.update

#bad
record.update_attributes!
return record if record.update_attributes
```

**Comment by bbatsov** [comment]
Something like this should be trivial to implement. It's a pity it won't work well in the other direction, but I guess it should be configurable.
